src/components/panes/ImportLogPane.scss

```diff
@@ -0,0 +1,88 @@
+.ImportLogPane {
+    .InfoListItem-status {
+        &:before {
+            @include icon($fa-var-question-circle);
```

Comment:
How about `fa-var-check` when successful, `fa-var-exclamation` if it failed, `fa-var-hourglass-half` if pending?